config/application.rb

```diff
@@ -10,6 +10,7 @@ module MentorPathway
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
+    config.assets.css_compressor = nil
```

Comment:
the config.assets.css_compressor line sets the CSS compressor that will be used to compress CSS assets. Setting the value to nil disables CSS compression. This means that the CSS files will not be compressed before they are served to the client. This can be useful for development or debugging but it is not recommended for production use, as it will increase the size of the CSS files and slow down page load times.